riptide-failsafe/README.md

```diff
@@ -0,0 +1,103 @@
+# Riptide: Failsafe
+
+[![Valves](../docs/valves.jpg)](https://pixabay.com/en/wheel-valve-heating-line-turn-2137043/)
+
+[![Build Status](https://img.shields.io/travis/zalando/riptide/master.svg)](https://travis-ci.org/zalando/riptide)
+[![Coverage Status](https://img.shields.io/coveralls/zalando/riptide/master.svg)](https://coveralls.io/r/zalando/riptide)
+[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/org.zalando/riptide-failsafe/badge.svg)](http://www.javadoc.io/doc/org.zalando/riptide-failsafe)
+[![Release](https://img.shields.io/github/release/zalando/riptide.svg)](https://github.com/zalando/riptide/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/org.zalando/riptide-failsafe.svg)](https://maven-badges.herokuapp.com/maven-central/org.zalando/riptide-failsafe)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/zalando/riptide/master/LICENSE)
+
+*Riptide: Failsafe* adds [Failsafe](https://github.com/jhalterman/failsafe) support to *Riptide*. It offers retries
+and a circuit breaker to every remote call.
+
+## Example
+
+```java
+Http.builder()
+    .plugin(new FailsafePlugin(scheduler)
+            .withRetryPolicy(retryPolicy)
+            .withCircuitBreaker(circuitBreaker))
+    .build();
+```
+
+## Features
+
+-  seamlessly integrates Riptide with Failsafe
+
+## Dependencies
+
+- Java 8
+- Riptide Core
+- Failsafe
+
+## Installation
+
+Add the following dependency to your project:
+
+```xml
+<dependency>
+    <groupId>org.zalando</groupId>
+    <artifactId>riptide-failsafe</artifactId>
+    <version>${riptide.version}</version>
+</dependency>
+```
+
+## Configuration
+
+The failsafe plugin will not perform retries nor apply circuit breakers unless they were explicitly configured:
+
+```java
+Http.builder()
+    .plugin(new FailsafePlugin(Executors.newScheduledThreadPool(20))
+            .withRetryPolicy(new RetryPolicy()
```

Comment:
Can we maybe extend the example with `@FailsafeBreaker` annotation and make use of failsafe-actuator here?

Reply:
I'd rather link to failsafe-actuator here (small paragraph after the example? any suggestions?) and properly support it in put-it-to-rest.

Reply:
Ok, seems legit. What about:

### Configuration with [Failsafe-Actuator](https://github.com/zalando-incubator/failsafe-actuator)

In case you are using [Failsafe-Actuator](https://github.com/zalando-incubator/failsafe-actuator) you can inject the Circuit Breaker as well.

```
Http.builder()
    .plugin(new FailsafePlugin(Executors.newScheduledThreadPool(20))
            .withRetryPolicy(new RetryPolicy()
                    .retryOn(SocketTimeoutException.class)
                    .withDelay(25, TimeUnit.MILLISECONDS)
                    .withMaxRetries(4))
            .withCircuitBreaker(@FailsafeBreaker("breakIt") breaker
                    .withFailureThreshold(3, 10)
                    .withSuccessThreshold(5)
                    .withDelay(1, TimeUnit.MINUTES)))
    .build();
```

Reply:
That annotation/injection looks incorrect.

Reply:
I added a section to the README based on your suggestion. Feel free to take a look.

Reply:
You are right, your changed version looks good!